### PR TITLE
EthGlobal Page Addition 

### PR DESCRIPTION
**Pull request by solanagirl**
Adding EthGlobal Page

https://www.figma.com/file/nG18Lmm7fXeuH7t3qofZDI/ETHGlobal?type=design&node-id=43-339&t=zdMGburaKWuvZu3n-0

### CODE REVIEW

File: app/public/white-star-icon.png
Comment (Vidhi-26) [inline app/public/white-star-icon.png:1]: delete this as it is not used

File: app/public/white-star.png
Comment (Vidhi-26) [inline app/public/white-star.png:1]: delete this as it is not used

**Review by Vidhi-26 — APPROVED** [review]
May delete unused images from public

**Review by markosg04 — APPROVED** [review]
LGTM for now. I have some suggestions for style but it's not important for this PR / repo. On the redesign we can enforce stricter style

**Pull request merged**